ceph-releases/luminous/ubuntu/16.04/daemon/clean_container.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+INITIAL_SIZE=$(du -sm / 2>/dev/null | awk {'print $1}')
+DEBUG=
+
+# Step 1
+# Let's remove easy stuff
+strip -s /usr/local/bin/{confd,forego,kubectl}
+strip -s /usr/bin/{crushtool,monmaptool,osdmaptool}
```

Comment:
I'm surprised `/usr/bin/{crushtool,monmaptool,osdmaptool}` is not stripped already. What OS / Ceph version were you using where this makes a difference?

Reply:
Get:131 http://download.ceph.com/debian-luminous xenial/main amd64 ceph-osd amd64 12.2.2-1xenial [12.9 MB]
Get:132 http://download.ceph.com/nfs-ganesha/deb-V2.5-stable/luminous xenial/main amd64 nfs-ganesha amd64 2.5.4-0-g9e93d73-1xenial [573 kB]